packages/fxa-shared/payments/configuration/manager.ts

```diff
@@ -74,8 +74,8 @@ export abstract class PaymentConfigManager {
    */
   public async load() {
     const [productResults, planResults] = await Promise.all([
-      this.productConfigDbRef.select().get(),
-      this.planConfigDbRef.select().get(),
+      this.productConfigDbRef.get(),
```

Comment:
I found that with `select()` as part of the request, the `doc.data()` a few lines down, wasn't actually returning any results.

Reply:
I'm still getting the same issue as before. After I added the `select()`'s back in `doc.data()` wasn't returning any data. 

I'll pair with someone from SubPlat tomorrow and see if they get the same issue.